retroshare.pri

```diff
@@ -501,8 +501,16 @@ no_rs_deprecatedwarning {
 }
 
 no_rs_cppwarning {
+    QMAKE_CXXFLAGS += -Wno-unknown-warning-option
     QMAKE_CXXFLAGS += -Wno-cpp
     QMAKE_CXXFLAGS += -Wno-inconsistent-missing-override
+    QMAKE_CXXFLAGS += -Wno-misleading-indentation
+    QMAKE_CXXFLAGS += -Wno-unused-parameter
```

Comment:
this one is very useful. Sometimes helps you detect some bugs.

Reply:
Actually all warnings below should stay. They *are* useful.

Reply:
So why you don't merge PR fixing it? ;)
For now all of them pollute compilation output.